api/src/main/java/no/bibsys/web/model/RegistryDto.java

```diff
@@ -75,8 +75,10 @@ public boolean equals(Object obj) {
                 && Objects.equals(schema, other.schema);
     }
 
-
-    
-    
+    @Override
+    public String toString() {
```

Comment:
Choose a name that expresses exactly what this is doing

Reply:
This overrides Object.toString()